examples/arduino/NotecardEnvVarManager.c

```diff
@@ -1 +0,0 @@
-../../src/NotecardEnvVarManager.c
```

Comment:
Why were the symlinks deleted?

Reply:
They're unnecessary in the Arduino example world.